test.disabled/ejabberd_tests/tests/adhoc_SUITE.erl

```diff
@@ -29,7 +29,7 @@ groups() ->
     [{adhoc, [sequence], [ping]}].
 
 suite() ->
-    escalus:suite().
+    [{timetrap,{seconds,30}}|escalus:suite()].
```

Comment:
Why exactly do we need these `timetrap`s in all suites? Could it be configured globally from the `default.spec` file?